ui/package.json

```diff
@@ -129,6 +129,6 @@
     "utf-8-validate": "^5.0.0"
   },
   "resolutions": {
-    "@mdx-js/runtime/**/acorn": "^6.4.0"
```

Comment:
It looks like @helfi92 added this resolution in #2135.  It looks like that was a "forced upgrade" from its listed dependency on 6.1.1.  I think we could remove the resolution now, as yarn-minify will round `^6.1.1` up to 6.4.2.

I don't think force-updating two major versions is a good idea!